pkgs/applications/version-management/git-and-tools/gh/default.nix

```diff
@@ -30,8 +35,10 @@ buildGoModule rec {
     done
   '';
 
-  # fails with `unable to find git executable in PATH`
-  doCheck = false;
+  checkInputs = [ git ];
+  checkPhase = ''
+    make test
+  '';
```

Comment:
Requires a custom ``checkPhase`` b/c it uses a custom ``buildPhase/installPhase``, so some of the shell functions (i.e. ``getGoDirs()`` that the ``buildGoModule.checkPhase`` expects are missing.